packages/utils/src/geoJson.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * @typedef {[number, number]} Point
+ */
+
+/**
+ * swap from lng lat to lat lng and keep longitude within [0, 360)
+ * @param {Point} coords
+ * @returns {Point}
+ */
+function flipLatLng(coords) {
+  const [lng, lat] = coords;
+  return [lat, (360 + lng) % 360];
+}
+
+/**
+ * Recursively calls flipLatLng
+ * @param {any[]} coords
+ * @returns {any[]}
+ */
+function flipLatLngRecursive(coords) {
+  if (typeof coords[0] === 'number') {
+    return flipLatLng(coords);
+  }
+
+  return coords.map(flipLatLngRecursive);
+}
+
+/**
+ * Convert point geojson string into [lat, lng]
+ * @param {string | null} pointGeoJsonString
+ * @returns {Point | null} [lat, lng]
+ */
+export function translatePoint(pointGeoJsonString) {
```

Comment:
Happy with these names or should they be a bit more verbose?
I feel the entire function signature pretty clearly explains what it does, however when it's imported and used inline it can be a little vague, eg:
```
    case 'coordinates': {
      const [lat, long] = translatePoint(entity.point);
      return `${lat}, ${long}`;
    }
```

Reply:
Inline that does look a bit vague. Nice and concise though, and could always look at the import to know it was coming from `geoJson`, which already gives you more info. 

I wonder if we should put helpers onto `EntityType`, so you can go `entity.getCoordinates();` or `entity.getBounds();` and it'll give you the nice format? That way there aren't many vague inline places